### PR TITLE
add iframe login methods

### PR DESCRIPTION
**Pull request by DanutIlie**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "@fortawesome/free-solid-svg-icons": "6.5.1",
     "@fortawesome/react-fontawesome": "0.2.0",
     "@multiversx/sdk-core": "13.15.0",
-    "@multiversx/sdk-dapp": "3.0.10",
```
Comment (arhtudormorar) [inline package.json:22]: 13?

File: src/context/index.tsx [hunk 2]
```diff
@@ -33,6 +33,10 @@ const ContextProvider = (props: PropsWithChildren) => {
           name: 'customConfig',
           apiTimeout: 6000,
           walletConnectV2ProjectId: '9b1a9564f91cb659ffe21b73d5c4e2d8',
+          metamaskSnapWalletAddress:
+            state.dappEnvironment === EnvironmentsEnum.mainnet
+              ? 'https://snap-wallet.multiversx.com'
```
Comment (arhtudormorar) [inline src/context/index.tsx:38]: what about testnet?

Comment (DanutIlie) [reply src/context/index.tsx]: i removed it, radu added defaults in skd-dapp

File: package.json (hunk 1 above)
Comment (DanutIlie) [reply package.json:22]: done

**Review by radumojic — APPROVED** [review]

**Review by arhtudormorar — APPROVED** [review]

**Pull request merged**